.dockerignore

```diff
@@ -2,6 +2,8 @@
 .github
 .gitignore
 .gitlab-ci.yml
+.rustfmt.toml
```

Comment:
Ignoring everything and the  un-ignoring things that are actually needed for build is typically a more productive approach than block-listing